prebackup/prebackup.go

```diff
@@ -219,6 +219,8 @@ func (p *PreBackup) waitForReady(watcher watch.Interface) error {
 			}
 			return fmt.Errorf("there was an unknown error while starting pre backup pod '%v/%v'", deployment.Namespace, deployment.Name)
 
+		case watch.Added:
+			// ignore this event, there would lots of them, but irrelevant.
```

Comment:
@Kidswiss do you think we should at least log the event with debug log level?

Reply:
Yeah why not, could be helpful for debugging stuff I guess.